### DISCUSSION TITLE
Can't understand error:not all sentences have tests

### DISCUSSION THREAD

**Discussion by Darigaf** [OHF-Voice/intents#1019] [_Q&A] [answered]
I was executing the terminal task to verify for errors on my changes and the error:
[ERROR] tests/pt-br/light_HassTurnOff.yaml: not all sentences have tests
appeared on my terminal and said the validation had failed. Does someone know what this error means? and how to fix it?

**Comment by al-one** [comment, marked as answer]
The length of the sentences tests must be greater than or equal to the length of the sentences.

https://github.com/home-assistant/intents/blob/3714d4df72caad2f23c3a77938c4baa689039340/script/intentfest/validate.py#L443-L456
